gax-grpc/src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -32,98 +32,70 @@
 import static com.google.longrunning.PagedResponseWrappers.ListOperationsPagedResponse;
 
 import com.google.api.core.ApiFunction;
-import com.google.api.core.ApiFuture;
 import com.google.api.core.BetaApi;
-import com.google.api.gax.core.GaxProperties;
 import com.google.api.gax.core.GoogleCredentialsProvider;
 import com.google.api.gax.core.InstantiatingExecutorProvider;
-import com.google.api.gax.grpc.GaxGrpcProperties;
-import com.google.api.gax.grpc.GrpcTransportChannel;
-import com.google.api.gax.retrying.RetrySettings;
-import com.google.api.gax.rpc.ApiCallContext;
 import com.google.api.gax.rpc.ApiClientHeaderProvider;
 import com.google.api.gax.rpc.ClientContext;
 import com.google.api.gax.rpc.ClientSettings;
-import com.google.api.gax.rpc.PageContext;
 import com.google.api.gax.rpc.PagedCallSettings;
-import com.google.api.gax.rpc.PagedListDescriptor;
-import com.google.api.gax.rpc.PagedListResponseFactory;
-import com.google.api.gax.rpc.StatusCode;
 import com.google.api.gax.rpc.UnaryCallSettings;
-import com.google.api.gax.rpc.UnaryCallable;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Lists;
-import com.google.longrunning.stub.GrpcOperationsStub;
 import com.google.longrunning.stub.OperationsStub;
+import com.google.longrunning.stub.OperationsStubSettings;
 import com.google.protobuf.Empty;
 import java.io.IOException;
 import javax.annotation.Generated;
-import org.threeten.bp.Duration;
 
 // AUTO-GENERATED DOCUMENTATION AND CLASS
 /** Settings class to configure an instance of {@link OperationsClient}. */
 @Generated("by GAPIC v0.0.5")
 @BetaApi
 public class OperationsSettings extends ClientSettings<OperationsSettings> {
-  private final UnaryCallSettings<GetOperationRequest, Operation> getOperationSettings;
-  private final PagedCallSettings<
-          ListOperationsRequest, ListOperationsResponse, ListOperationsPagedResponse>
-      listOperationsSettings;
-  private final UnaryCallSettings<CancelOperationRequest, Empty> cancelOperationSettings;
-  private final UnaryCallSettings<DeleteOperationRequest, Empty> deleteOperationSettings;
+
+  public static final OperationsSettings create(OperationsStubSettings stub) throws IOException {
+    return new OperationsSettings.Builder(stub.toBuilder()).build();
+  }
```

Comment:
Put this down with the other static methods.

Reply:
Done.